### ISSUE TITLE
JDBC/DBeaver - impossible to view table with CREATE TYPE enum

### ISSUE THREAD

**Issue by 2-5** [duckdb/duckdb#7537]
### What happens?

DBeaver fails to display a table with `CREATE TYPE` columns.

### To Reproduce

Create `db_type.duckdb`:

```python
import duckdb

db = duckdb.connect("db_type.duckdb")
db.execute("CREATE TYPE direction AS ENUM ('up', 'down')")
db.execute("CREATE TABLE tbl (id INTEGER, dir direction)")
db.execute("INSERT INTO tbl VALUES (1, 'up')")
db.execute("INSERT INTO tbl VALUES (2, 'down')")
db.close()
```

Open it in DBeaver and try to view the `tbl` table:

```
org.jkiss.dbeaver.DBException: No enum constant org.duckdb.DuckDBColumnType.direction
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:249)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:121)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:5096)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.lang.IllegalArgumentException: No enum constant org.duckdb.DuckDBColumnType.direction
	at java.base/java.lang.Enum.valueOf(Unknown Source)
	at org.duckdb.DuckDBColumnType.valueOf(DuckDBColumnType.java:3)
	at org.duckdb.DuckDBResultSetMetaData.TypeNameToType(DuckDBResultSetMetaData.java:58)
	at org.duckdb.DuckDBDatabaseMetaData.getColumns(DuckDBDatabaseMetaData.java:827)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCDatabaseMetaDataImpl.getColumns(JDBCDatabaseMetaDataImpl.java:1152)
	at org.jkiss.dbeaver.ext.generic.model.meta.GenericMetaModel.prepareTableColumnLoadStatement(GenericMetaModel.java:708)
	at org.jkiss.dbeaver.ext.generic.model.TableCache.prepareChildrenStatement(TableCache.java:82)
	at org.jkiss.dbeaver.ext.generic.model.TableCache.prepareChildrenStatement(TableCache.java:1)
	at org.jkiss.dbeaver.model.impl.jdbc.cache.JDBCStructCache.loadChildren(JDBCStructCache.java:96)
	at org.jkiss.dbeaver.model.impl.jdbc.cache.JDBCStructCache.getChildren(JDBCStructCache.java:221)
	at org.jkiss.dbeaver.ext.generic.model.GenericTableBase.getAttributes(GenericTableBase.java:188)
	at org.jkiss.dbeaver.model.impl.jdbc.struct.JDBCTable.readRequiredMeta(JDBCTable.java:912)
	at org.jkiss.dbeaver.model.impl.jdbc.struct.JDBCTable.readData(JDBCTable.java:134)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:123)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:189)
	... 4 more
```

### OS:

Windows

### DuckDB Version:

0.7.1

### DuckDB Client:

JDBC duckdb_jdbc-0.7.1.jar

### Full Name:

2-5

### Affiliation:

private

### Have you tried this on the latest `master` branch?

- [ ] I agree

### Have you tried the steps to reproduce? Do they include all relevant data and configuration? Does the issue you report still appear there?

- [X] I agree

**Comment by Mause** [comment]
This has been fixed already in #7000, can you please try the dev version, or the 0.8.0 version when it's released? (should be released this week sometime)

**Comment by 2-5** [comment]
Confirmed 0.8.0 fixed it
